js/index.js

```diff
@@ -38,6 +38,15 @@ btnAct[1].addEventListener('mouseout', (e3) => {
 btnAct[2].addEventListener('mouseout', (e3) => {
     e3.target.innerHTML ="Sign Me Up!";
 })
```

Comment:
Instead of just changing inner HTML try to make a whole new event listener that does something completely random, make it change colors, make it grow, make it shoot off screen, something like that.